en/advanced_config/flight_termination.md

```diff
@@ -0,0 +1,47 @@
+# Flight Termination Configuration
+
+The *Flight termination* [failsafe action](../config/safety.md#failsafe_actions) may be triggered by a [safety check](../config/safety.md) (e.g. RC Loss, geofence violation, etc. on any vehicle type or in any flight mode), or by the [FailureDetector](../config/safety.md#failure_detector).
+
+When *Flight termination* is activated, PX4 simultaneously turns off all controllers and sets all PWM outputs to their failsafe values.
+
+Depending on what devices you have connected the PWM failsafe outputs can be used to:
+- Deploy a [parachute](../peripherals/parachute.md).
+- Extend retractable landing gear.
+- Move a PWM-connected gimbal to a safe orientation (or retract it) in order to protect the camera.
+- Trigger an inflatable device like an airbag.
+- Trigger an alarm.
+
+There is no way to recover from flight termination.
+After triggering you should unplug the battery as soon as possible
+You will need to reboot/power cycle the vehicle before it can be used again.
+
+> **Tip** PX4 does not know what devices are attached - it just applies a predefined set of PWM values to its outputs.
+  This all happens at the same time - there is no way to configure independent time-based (or other) triggering of the motors or specific safety devices.
+  
+<span></span>
+> **Note** This is *not* an independent Flight Termination System. If power is lost or if the autopilot crashes completely, the failsafe devices will not triggered.
+
+
+## Hardware Configuration
+
+You can use any *safety device(s)* you like that can be triggered by changing a PWM value.
+These can be connected to any free PWM port (both MAIN and AUX).
+
+> **Note** If you're using Pixhawk-series board you will have to separately power the servo rail (i.e. from a 5V BEC, which is often also available from your ESC).
+
+
+## Software Configuration
+
+The [Safety](../config/safety.md) topic explains how to set *flight termination* as the action to be performed in the event of particular checks being triggered (this also covers the parameters you need to set to trigger the [Failure Detector](../config/safety.md#failure_detector), which always results in *Flight termination*).
+
+- Set [CBRK_FLIGHTTERM=0](../advanced_config/parameter_reference.md#CBRK_FLIGHTTERM) to enable flight termination. 
```

Comment:
Note that this only cuts FailureDetector and FMU loss features. The other flight terminations (RC, telem, geofence) are not disabled by that circuit breaker.

Reply:
Made this more clear